front/src/common/Map/Buttons/MapButtons.tsx

```diff
@@ -65,14 +65,53 @@ export default function MapButtons({
 
   useOutsideClick(mapButtonsRef, () => setOpenedPopover(undefined));
 
+  const ZOOM_DEFAULT = 5;
+  const ZOOM_DELTA = 1.5;
+  const DEFAULT_VIEWPORT = {
+    latitude: 47.3,
+    longitude: 2.0,
+    zoom: 5.0,
+    bearing: 0,
+    pitch: 0,
+  };
```

Comment:
You can move all these const outside MapButtons function or even in an other file :) otherwise these const will be re-instantiated at each re-render
```suggestion
```